src/CommandLine/Parser.ts

```diff
@@ -0,0 +1,55 @@
+import { CommandBase } from './Commands/Base';
+import WriteCommand from './Commands/Write';
+import WallCommand from './Commands/WriteAll';
+import QuitCommand from './Commands/Quit';
+import QuitAllCommand from './Commands/QuitAll';
+import WriteQuitCommand from './Commands/WriteQuit';
+import WriteQuitAllCommand from './Commands/WriteQuitAll';
+import VisualSplitCommand from './Commands/VisualSplit';
+import NewFileCommand from './Commands/NewFile'
+import GoToLineCommand from './Commands/GoToLine';
+
+const commandParsers = {
+    w: WriteCommand,
+    write: WriteCommand,
+    wa: WallCommand,
+    wall: WallCommand,
+
+    q: QuitCommand,
+    quit: QuitCommand,
+    qa: QuitAllCommand,
+    qall: QuitAllCommand,
+
+    wq: WriteQuitCommand,
+    writequit: WriteQuitCommand,
+    x: WriteQuitCommand,
+
+    wqa: WriteQuitAllCommand,
+    wqall: WriteQuitAllCommand,
+    xa: WriteQuitAllCommand,
+    xall: WriteQuitAllCommand,
+
+    vs: VisualSplitCommand,
+    vsp: VisualSplitCommand,
+    sp: VisualSplitCommand,
+    split: VisualSplitCommand,
```

Comment:
`sp` and `split` are for the horizontal split. VSCode doesn't support this right now.

Reply:
VSCode should support horizontal split now